features/storage/kvstore/filesystemstore/FileSystemStore.cpp

```diff
@@ -0,0 +1,627 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
```

Comment:
Can you add SPDX identifier to new files (we should be using them as written in the license docs). Old files are getting updated slowly

Also should be `Arm` instead of `ARM`

Reply:
@0xc0170 - sure, do you mean something like:

* Copyright (c) 2018, Arm Limited and affiliates.
* SPDX-License-Identifier: Apache-2.0

Reply:
Added where missing.